<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-62P709O" failures="4" tests="5" name="Test(failed)" time="75.517" errors="0" timestamp="2024-10-07T17:38:45 IST">
  <testcase classname="testCases.TC002_LoginTestcase" name="Login" time="3.113"/>
  <testcase classname="testCases.TC003_DataDrivenTesting" name="verify_DDT" time="11.903">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_DataDrivenTesting.verify_DDT(TC003_DataDrivenTesting.java:76)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_DDT -->
  <testcase classname="testCases.TC003_DataDrivenTesting" name="verify_DDT" time="17.358">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_DataDrivenTesting.verify_DDT(TC003_DataDrivenTesting.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_DDT -->
  <testcase classname="testCases.TC003_DataDrivenTesting" name="verify_DDT" time="13.123">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_DataDrivenTesting.verify_DDT(TC003_DataDrivenTesting.java:76)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_DDT -->
  <testcase classname="testCases.TC003_DataDrivenTesting" name="verify_DDT" time="11.211">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_DataDrivenTesting.verify_DDT(TC003_DataDrivenTesting.java:76)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_DDT -->
</testsuite> <!-- Test(failed) -->
